google-cloud-spanner/src/main/java/com/google/cloud/spanner/GrpcStruct.java

```diff
@@ -460,6 +468,12 @@ protected String getPgJsonbInternal(int columnIndex) {
     return (String) rowData.get(columnIndex);
   }
 
+  @Override
+  protected long getPgOidInternal(int columnIndex) {
```

Comment:
remove and rely on `getLongInternal` instead

Reply:
Done.